### DISCUSSION TITLE
Navbar Customizer

### DISCUSSION THREAD

**Discussion by VaibhavSys** [Paymenter/Paymenter#387] [Ideas]
A built-in navbar customizer would be of great benefit and would allow merchants to add the panel link and other things in the navbar easily and in a clean manner.

**Comment by SirHoDo** [comment]
> A built-in navbar customizer would be of great benefit and would allow merchants to add the panel link and other things in the navbar easily and in a clean manner.

There is already a built-in navbar customizer called the themes folder. Themes are very simple to setup and edit

**Reply by VaibhavSys** [reply]
If we edit the theme files directly, the update process of the theme will not be as straightforward then, the edits will need to be reapplied. A built-in navbar customizer would solve this in a clean manner.

**Reply by Emeryoutdoors** [reply]
If you are talking about adding buttons im all in it ngl. Much needed
